package.json

```diff
@@ -1,14 +1,18 @@
 {
   "name": "@openstax/highlighter",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "main": "dist/highlighter.js",
   "license": "MIT",
   "scripts": {
-    "typescript-watch": "tsc --noEmit --watch",
-    "webpack-watch": "webpack --config=webpack.config.js --mode=development --watch",
     "build": "webpack --config=webpack.config.js --mode=production",
     "jest-watch": "jest --watchAll",
-    "watch": "npm-run-all jest-watch webpack-watch typescript-watch"
+    "test": "jest --verbose --maxWorkers=2",
+    "type-check": "tsc",
+    "js-lint": "eslint src/**/*.js",
```

Comment:
What do you think of having 1 lint command that calls all the lint subcommands? Something like https://github.com/philschatz/repo-template/blob/master/package.json#L12

I used `:` rather than `-` to denote subcommands so that `-` could be used for multi-word commands